create_db.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE CONTACTS ( [id] INTEGER PRIMARY KEY, 
```

Comment:
Table names should be singular.
please adjust this.
https://softwareengineering.stackexchange.com/questions/290646/why-does-convention-say-db-table-names-should-be-singular-but-restful-resources